src/core.js

```diff
@@ -368,11 +367,6 @@ olcs.core.tileLayerToImageryLayer = function(olLayer, viewProj) {
   let provider = null;
 
   const source = olLayer.getSource();
-  // handle special cases before the general synchronization
-  if (source instanceof ol.source.WMTS) {
```

Comment:
Please modify the changelog to state we now handle WMTS sources, expecting the tilegrid is the same.